coreengine/audiomanager.cpp

```diff
@@ -577,7 +570,14 @@ void AudioManager::SlotAddMusic(QString file, qint64 startPointMs, qint64 endPoi
         }
         else
         {
-            CONSOLE_PRINT("Unable to locate music file: " + currentPath, GameConsole::eERROR);
+            CONSOLE_PRINT("Unable to locate music file: " + file, GameConsole::eERROR);
```

Comment:
Please change this so it's not using recursion aka with a function for loading which returns a bool.
To try loading the files which gets called.
Recursion is never a good way for stable code.